tests-utils/src/main/scala/tech/beshu/ror/utils/containers/generic/EsClusterContainer.scala

```diff
@@ -131,14 +131,13 @@ trait RemoteClustersInitializer {
 }
 
 final case class EsClusterSettings(name: String,
-                                   rorConfigFileName: String,
                                    numberOfInstances: Int = 1,
                                    nodeDataInitializer: ElasticsearchNodeDataInitializer = NoOpElasticsearchNodeDataInitializer,
                                    clusterInitializer: EsClusterInitializer = NoOpEsClusterInitializer,
                                    dependentServicesContainers: List[DependencyDef] = Nil,
                                    xPackSupport: Boolean = false,
                                    configHotReloadingEnabled: Boolean = true,
                                    customRorIndexName: Option[String] = None,
-                                   internodeSslEnabled: Boolean = false)
+                                   internodeSslEnabled: Boolean = false)(implicit val rorConfigFileName: String)
```

Comment:
why it's passed as implicit? how is it different from other settings?

Reply:
We don't have to do:
```scala

override val rorConfigFileName = "/cluster_state_internode_ssl/readonlyrest.yml"
    
EsClusterSettings(
      ...
      rorConfigFileName = rorConfigFileName
      ...
)
```
 for each test. Only define config file name. IMO suite looks cleaner